ironic-deployment/ironic/ironic.yaml

```diff
@@ -104,6 +90,14 @@ spec:
           envFrom:
             - configMapRef:
                 name: ironic-bmo-configmap
+        - name: httpd-reverse-proxy
+          image: quay.io/metal3-io/ironic-inspector
```

Comment:
Should it be inspector image or ironic image?

Reply:
I think I got the answer. It was added here https://github.com/metal3-io/ironic-inspector-image/commit/7e2757d890917dc7256bbae1b23fed00c8e74b73#diff-9f9f6a924db88783257f0da0c2ae75be9d01b5b1ea71d522caaafab42362392f